src/components/Menu/Menu.stories.tsx

```diff
@@ -442,6 +442,36 @@ export const MenuHighlightColors: Story = {
         selectedItem={"item-title-1"}
         listItemsData={defaultListItems}
       />
+      <Menu
+        labelText={"Schomburg"}
```

Comment:
Nit: You can pass the string directly without the braces.

Reply:
Question about this.  If brackets are not needed for strings, why does does VS Code add them by default?